### PR TITLE
Snow emergency status 40

### LINKED ISSUE #40: Intent: Snow Emergency
User should be able to ask if there is currently a snow emergency in the city.

There is no API for this information, but it is posted at http://www.boston.gov/snow as a banner at the top of this page. We should be able to parse this page. For development, lets see if we can see how this works from the page source as well as https://github.com/CityOfBoston/boston.gov

### PR DESCRIPTION
**Pull request by jmartini**
Add a way to ask for snow or weather related alerts. I am assuming these will be info in the alert banner from @wesdrew 's work on all city alerts. We should double check this on the next snow emergency as well as what other key works should go in `SNOW_ALERT_QUERRY`

Closes #40

### CODE REVIEW

File: mycity/mycity/intents/get_alerts_intent.py [hunk 1]
```diff
@@ -58,6 +58,9 @@ class Services(Enum):
 TOW_LOT_NORMAL_MESSAGE += "Automated kiosks are available 24 hours a day, "
 TOW_LOT_NORMAL_MESSAGE += "seven days a week for vehicle releases."
 
+# Strings we use to determine if there is snow related alert
+SNOW_ALERT_QUERRY = ["snow", "winter weather", "inclement weather"]
```
Comment (wesdrew) [inline mycity/mycity/intents/get_alerts_intent.py]: change to `SNOW_ALERT_QUERY`

File: mycity/mycity/intents/get_alerts_intent.py [hunk 2]
```diff
@@ -76,20 +79,56 @@ def get_alerts_intent(
...
-    mycity_response.card_title = ALERTS_INTENT_CARD_TITLE
-    mycity_response.reprompt_text = None
+    mycity_response = _create_response_object()
     mycity_response.output_speech = alerts_to_speech_output(pruned_alerts) \
         if alerts_to_speech_output_function_for_test is None else alerts_to_speech_output_function_for_test(pruned_alerts)
+    return mycity_response
+
+
+def get_inclement_weather_alert(
+    mycity_request: MyCityRequestDataModel,
```
Comment (wesdrew) [inline mycity/mycity/intents/get_alerts_intent.py]: Nice, type-hinting is cool

File: mycity/mycity/intents/get_alerts_intent.py [hunk 3]
```diff
@@ -76,20 +79,56 @@ def get_alerts_intent(
...
+    """
+    logger.debug('MyCityRequestDataModel received:' + mycity_request.get_logger_string())
+
+    alerts = get_alerts() if get_alerts_function_for_test is None else get_alerts_function_for_test()
+    logger.debug("[dictionary with alerts scraped from boston.gov]:\n" + str(alerts))
+
+    logger.debug("filtering for inclement weather alerts")
+    output_speech = constants.NO_INCLEMENT_WEATHER_ALERTS
+    if Services.ALERT_HEADER.value in alerts:
+        if any(querry in alerts[Services.ALERT_HEADER.value].lower() for querry in SNOW_ALERT_QUERRY):
```
Comment (wesdrew) [inline mycity/mycity/intents/get_alerts_intent.py]: `querry` --> `query`

File: mycity/mycity/intents/speech_constants/get_alerts_intent.py [hunk 4]
```diff
@@ -4,3 +4,4 @@
 """
 
 NO_ALERTS = "There are no alerts. City services are running on normal schedules."
+NO_INCLEMENT_WEATHER_ALERTS = "There are currently no weather related alerts"
```
Comment (wesdrew) [inline mycity/mycity/intents/speech_constants/get_alerts_intent.py]: Nitpick: `"There are no weather related alerts."` scans better for me. Ignore if you want

File: mycity/mycity/test/unit_tests/test_get_alerts_intent.py [hunk 5]
```diff
@@ -67,3 +67,38 @@ def test_that_prune_normal_responses_returns_an_empty_dictionary_if_all_response
...
+        response = get_alerts_intent.get_inclement_weather_alert(
+            self.request,
+            self.stub_get_alerts)
+        self.assertEqual(constants.NO_INCLEMENT_WEATHER_ALERTS, response.output_speech)
+
+    def test_inclement_weather_alert_returns_normal_response_for_keyword_in_wrong_alert_section(self):
+        self.get_alerts_stub_return_dictionary = {
+            get_alerts_intent.Services.PARKING_METERS: "Snow doesn't bother us!"
+        }
+        response = get_alerts_intent.get_inclement_weather_alert(
```
Comment (wesdrew) [inline mycity/mycity/test/unit_tests/test_get_alerts_intent.py]: Nitpick: Do these function calls need to be split across three lines? Does PEP complain? If so, ignore

Comment (jmartini) [reply mycity/mycity/test/unit_tests/test_get_alerts_intent.py]: They do not, this is habit from code style guidelines used at iZotope. I'll make it one line.

**Comment by jmartini** [comment]
@wesdrew ready for another look

**Pull request merged**